features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/blocks/loganalyzer-dashboard/ajax/loganalyzer-dashboard.jag

```diff
@@ -0,0 +1,13 @@
+<%
+include("/jagg/jagg.jag");
```

Comment:
Hi Tharik,

This file is no longer use and we have removed this by the commits in : https://github.com/wso2/carbon-apimgt/pull/2429

Thanks
SajithD